scripts/scil_analyse_lesions_load.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This script will output informations about lesion load in bundle(s).
+The input can either be streamlines, binary bundle map, or a bundle voxel
+label map.
+"""
+
+import argparse
+import json
+import os
+
+import nibabel as nib
+import numpy as np
+import scipy.ndimage as ndi
+
+
+from scilpy.io.image import get_data_as_mask, get_data_as_label
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             add_json_args,
+                             assert_outputs_exist,
+                             add_reference_arg)
+from scilpy.segment.streamlines import filter_grid_roi
+from scilpy.tractanalysis.streamlines_metrics import compute_tract_counts_map
+from scilpy.utils.filenames import split_name_with_nii
+from scilpy.utils.metrics_tools import compute_lesion_stats
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_lesion',
+                   help='Binary mask of the lesion (.nii.gz).')
+    p.add_argument('out_json',
+                   help='Output file (.json) for lesion information')
+    p1 = p.add_mutually_exclusive_group()
+    p1.add_argument('--bundle',
+                    help='Path of the bundle file (.trk).')
+    p1.add_argument('--bundle_mask',
+                    help='Path of the bundle binary mask.')
+    p1.add_argument('--bundle_labels_map',
+                    help='Path of the bundle labels_map.')
+
+    p.add_argument('--min_lesion_vol', type=float, default=7,
+                   help='Minimum lesion volume in mm3 [%(default)s].')
+    p.add_argument('--out_lesion_atlas', metavar='FILE',
+                   help='Save the lesion as an atlas.')
```

Comment:
Not quite sure what this sentence means at first read. i.e. "lesion as an atlas" 
What does that mean?